templates/feature/app/scripts/packs/graphic.js

```diff
@@ -12,6 +12,10 @@ const clearContainer = () => container.html('');
 // a helper function to grab the container's width
 const getFrameWidth = () => container.node().offsetWidth;
 
+// import data by getting the window variable, OR by importing the filepath
+// let data = window.DATA;
+// import data from '../../../data/data.json';
```

Comment:
👋 Hello! The path to the data folder is aliased here in scripts: https://github.com/texastribune/data-visuals-create/blob/59182b355efaffee55547b8fccd465c10156e973/templates/__common__/config/webpack-utils.js#L182

This means you can also do this and it’d resolve correctly.

```js
import data from 'Data/data.json';
```

Reply:
WOW I DID NOT KNOW THAT. thanks for pointing that out!